sdk/identity/identity-persistence/samples-dev/basic.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license
+
+import { useIdentityExtension, DeviceCodeCredential, AccessToken } from "@azure/identity";
+
+import persistence from "@azure/identity-persistence";
+
+useIdentityExtension(persistence);
+
+import dotenv from "dotenv";
+dotenv.config();
+
+async function logToken(getToken: () => Promise<AccessToken>) {
+  console.log((await getToken()).token);
+}
+
+async function main() {
+  const credential = new DeviceCodeCredential({ tokenCachePersistenceOptions: {} });
+
+  const getToken = () => credential.getToken("https://witemple-appconfig.azconfig.io/.default");
+
+  console.log(1);
```

Comment:
I love this sample! However, I don’t think this console.log is doing much

Reply:
They are merely visual indicators to separate the walls of base64 in the console. These samples are not done, only used for my purposes in testing.